astro/src/content/docs/lifecycle/manage-users/tenant-manager/index.mdx

```diff
@@ -0,0 +1,109 @@
+
+### Creating Users
+To create a user, navigate to the Users view and click `Create`. Enter the new user account details on the New User view that opens.
+
+The `Basic information` section contains a form for setting user properties.  To customize the properties in the Basic information section:
+* Open the FusionAuth Admin application
+* Open <Breadcrumb>Customizations > Forms</Breadcrumb> 
+* Create a new `Admin Registration` form
+* Open the `Tenant manager` application
+* Select your new form in the <Breadcrumb>Registration > Admin Registration</Breadcrumb> setting
+
+### Editing Users
+From the Users view, clicking on a user takes you to the Edit User view. From here, you can edit the user's information, change their avatar, or take actions against their account.
+
+![Users View](/img/docs/lifecycle/manage-users/tenant-manager/edit-view.png)
+
+As with creating users, the fields you see in the <Breadcrumb>Edit</Breadcrumb> view come from the `Admin Registration` form that is set on the application.
+
+Note that you cannot set or change a user's password using the Tenant Manager app. If a user needs a new password, use one of the password action items described below.
+
+#### Actions
+From the <Breadcrumb>Edit</Breadcrumb> page, you can take various actions on a user.
+* The <InlineUIElement>Lock user</InlineUIElement> or <InlineUIElement>Unlock user</InlineUIElement> button will lock or unlock the user being viewed. A locked user is unable to log in to any application.
+* The menu next to the <InlineUIElement>Lock user</InlineUIElement> button provides additional actions
+  * Change password on next login: The user will be prompted for a new password the next time they log in
+  * Send password change email: This is useful for a user that has forgotten their password. An email will be sent to the email set on `user.email` or `user.data.email`, allowing the user to follow a link to a password change page. 
+  * Delete user: This will irreversibly delete the user's account, and should be done with caution. Locking a user's account is often a better, non-destructive option.
+
+## Roles and Permissions
+By default, a user with access to the Tenant Manager app will have read-only access. They will be able to search and view accounts, but will be unable to take any actions against user accounts.
+
+The `admin` role grants a user full permissions in the Tenant Manager app.
+
+You can assign roles to a user by managing their registration to the Tenant Manager app (see above).
+
+| Action | Standard user | Admin user |
+| -- | :--: | :--: |
+| List users | Yes | Yes |
+| Create user | No | Yes |
+| Edit user | No | Yes |
+| Delete user | No | Yes |
+| Lock / unlock user | No | Yes |
+| Force password change on next login| No | Yes |
+| Send password change email | No | Yes |
+| Delete user| No | Yes |
+
+
+## Customization
+At this time, the Tenant Manager app is not themable. 
```

Comment:
Not technically true. The admin user form for a tenant is customizable, on the tenant configuration, and so the fields that show up here can be changed by the configuration on the tenant.

Additionally, any custom data fields will resolve their name from a theme. So, if I wanted to add `user.data.favoriteColor` I would need to add `user.data.favoriteColor=Favorite color` to a theme message bundle associated with that tenant.

Reply:
Added.